.github/workflows/build-40.yml

```diff
@@ -1,15 +1,32 @@
 name: ublue main 40
+
 on:
   pull_request:
   merge_group:
   schedule:
     - cron: '5 3 * * *'  # 3am-ish UTC everyday (timed against official fedora container pushes)
   workflow_dispatch:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
+  cancel-in-progress: true
+
 jobs:
   build:
     name: build
     uses: ./.github/workflows/reusable-build.yml
     secrets: inherit
+    strategy:
+      fail-fast: false
+      matrix:
+        image_name:
+          - base
+          - silverblue
+          - kinoite
+          - sericea
+          - onyx
+          - lazurite
+          - vauxite
     with:
```

Comment:
Once F42 is out, we can simply duplicate this workflow and remove the images we don't wish to build